docs/book/src/developers/jobs.md

```diff
@@ -0,0 +1,102 @@
+# Jobs
+
+This document provides an overview of our jobs running via Prow and Github actions.
+
+## Builds and Tests running on the default branch
+
+
+ > NOTE: To see which test jobs execute which e2e tests, you can click on the links which lead to the
+  respective test overviews in testgrid.
+
+
+### Legend
+
+    🟢 REQUIRED - Jobs that have to run successfully to get the PR merged.
+
+
+### Presubmits
+
+  Prow Presubmits:
+
+  - 🟢 [pull-cluster-api-provider-azure-build]  `./scripts/ci-build.sh`
+  - 🟢 [pull-cluster-api-provider-azure-test]  `./scripts/ci-test.sh` 
+  - 🟢 [pull-cluster-api-provider-azure-e2e] ` ./scripts/ci-e2e.sh`
+       * GINKGO_FOCUS:`Workload cluster creation` GINKGO_SKIP:`Creating a GPU-enabled cluster|.*Windows.*|.*AKS.*|Creating a cluster that uses the external cloud provider`
+  - 🟢 [pull-cluster-api-provider-azure-windows] `./scripts/ci-e2e.sh`
+       * GINKGO_FOCUS:`.*Windows.*`
+  - 🟢 [pull-cluster-api-provider-azure-verify]   `make verify`     
+  - [pull-cluster-api-provider-azure-e2e-exp]  `./scripts/ci-e2e.sh`
+       * GINKGO_FOCUS:`.*AKS.*`
+  - [pull-cluster-api-provider-azure-apidiff]  `./scripts/ci-apidiff.sh`
+  - [pull-cluster-api-provider-azure-coverage]  `./scripts/ci-test-coverage`
+  - [pull-cluster-api-provider-e2e-full]  `./scripts/ci-e2e.sh`
+       * GINKGO_FOCUS:`Workload cluster creation`
+  - [pull-cluster-api-provider-capi-e2e]  `./scripts/ci-e2e.sh`
+       * GINKGO_FOCUS:`Cluster API E2E tests`
+  - [pull-cluster-api-provider-azure-conformance-v1alpha4]  `./scripts/ci-conformance.sh`
+  - [pull-cluster-api-provider-azure-upstream-v1alpha4-windows]  `./scripts/ci-conformance.sh`
+       * WINDOWS:`true` CONFORMANCE_NODES:`4`
+  - [pull-cluster-api-provider-azure-conformance-with-ci-artifacts]  `./scripts/ci-conformance.sh`
+       * E2E_ARGS:`-kubetest.use-ci-artifacts`
+  - [pull-cluster-api-provider-azure-windows-upstream-with-ci-artifacts]  `./scripts/ci-conformance.sh`
+       * E2E_ARGS:`-kubetest.use-ci-artifacts` WINDOWS:`true` CONFORMANCE_NODES:`4`
+
+
+  Github Presubmits Workflows:
+
+  - Markdown-link-check  `find . -name \*.md | xargs -I{} markdown-link-check -c .markdownlinkcheck.json {}`
+
+
+### Postsubmits
+  
+  Prow Postsubmits:
+  
+  - [post-cluster-api-provider-azure-push-images]  `/run.sh`
+       * args: 
+           - project=`k8s-staging-cluster-api-azure`
+           - scratch-bucket=`gs://k8s-staging-cluster-api-azure-gcb`
+           - env-passthrough=`PULL_BASE_REF`
+  - [postsubmits-cluster-api-provider-azure-e2e-full-main]  `runner.sh` , `./scripts/ci-e2e.sh`
```

Comment:
we don't need runner.sh here, the command to run it locally is `GINKGO_FOCUS="Workload cluster creation" ./scripts/ci-e2e.sh`